packages/core/src/common/path.ts

```diff
@@ -230,6 +249,20 @@ export class Path {
         return this.raw;
     }
 
+    /**
+     * Converts the current path into a file system path.
+     * @param posix Indicates posix or windows file path.
+     * If `undefined`, the format will be determined by the `OS.backend.type` value.
+     * @returns A file system path.
+     */
+    fsPath(posix?: boolean): string {
```

Comment:
What would you think of an enum here? A bit more typing, but more explicit than the boolean.

Reply:
I agree, I introduced an enum for this 👍